File: Common Library/DotNet_Framework/PredictorAPI/Models/ProbaWinner.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PredictorAPI.Models
{
    public class ProbaWinner
    {
        [JsonProperty("win_proba")]
        public double WinProba { get; set; }

        [JsonProperty("draw_proba")]
        public double DrawProba { get; set; }

        [JsonProperty("lose_proba")]
        public double LoseProba { get; set; }
    }
}

File: MachineLearning/Poseidon_Predictor_Server/app/main/model/proba_winner.py
class ProbaWinner():
    def __init__(self):
        self.win_proba = None
        self.draw_proba = None
        self.lose_proba = None
    
    def SetWinProba(self, win_proba):
        self.win_proba = win_proba

    def SetDrawProba(self, draw_proba):
        self.draw_proba = draw_proba

    def SetLoseProba(self, lose_proba):
        self.lose_proba = lose_proba